x-pack/plugin/sql/qa/server/src/main/resources/version.csv-spec

```diff
@@ -0,0 +1,69 @@
+// To mute tests follow example in file: example.csv-spec
+
+//
+// Tests for VERSION fields
+//
```

Comment:
a couple more things that I would cover:

* group by version
* constant expression returning a version (cast)
* select a string field casted to a version (script returning a version value). like `select concat(str, '.0')::version from "versions"`
* order by script returning a version
* group by script returning a version
* cast a version to a string

Reply:
Thank you @Luegg, it revealed a problem:

```
select '1.2.3'::version < '1.11.0'::version
```

the comparison takes into consideration SemVer order, we will have to reimplement it in *QL (now it's just comparing strings, that is wrong)

Reply:
Just for the record, the same happens with `ip` field type

Reply:
oh, I didn't see your response. I also created an issue for the IP case https://github.com/elastic/elasticsearch/issues/85563